plugins/services/src/js/components/modals/ServiceGroupFormModal.js

```diff
@@ -37,11 +37,28 @@ class ServiceGroupFormModal extends React.PureComponent {
   }
 
   getErrorMessage() {
-    const { errors } = this.props;
+    const { errors, i18n } = this.props;
     if (!errors) {
       return null;
     }
 
+    // The server error message for duplicate group name looks like
+    // "Group /name is already created. Use PUT to change this group."
+    // So we check the second sentence and if it matches
+    // we add our custom translatable error message.
+    if (
+      errors.split(".")[errors.split(".").length - 2] ===
```

Comment:
I realize that this code is ugly, but for me the benefits of having a understandable and translatable error message outweighs the uglyness of the code. If two reviewers disagree with me, I will change it.